### ISSUE TITLE
Should not receive notification about closing poll I already voted in

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#6113]
I just got an email reminding me that a pool is closing, so I opened the site, only to find that I had already voted.

Shouldn't loomio skip sending me a notification in this case? Or at least let me opt out of such messages, given the vast number of messages we all have to wade through already?

Every single one of them makes it more unlikely that I'll open the next one, especially if the message just turns out to be pointless.

**Comment by robguthrie** [comment]
Some poll types have an option "voters_review_responses" which controls this behaviour.
The idea is that some processes involve people reviewing how everyone voted before it closes to see if they want to change their position.

**Comment by madduck** [comment]
@robguthrie yeah makes sense, but obviously not for polls like "I have read and am ready". Ideal would be a check box next to the vote button "keep me posted about this poll".

**Comment by robguthrie** [comment]
nice. What kind of poll was this?

**Comment by madduck** [comment]
So this was a time poll, and I am torn on whether it makes sense for me to be invited to look again. Maybe my calendar did change? Maybe I'll feel more motivated moving things around, if I take the time to understand that changing my vote would facilitate the decision? Sure.

However, in this case, I better be honest with you: I didn't even read the notification. I went to the site, saw that I had already voted (without even checking what the vote was), and went on to file this bug report, thinking that it was a yes/no poll about signing off "the minutes that reflect the board meeting".

So yeah, bonus points to those who read before wasting someone else's time (your time), but I guess it kinda emphasises the point: too much email is bad.
